hq/app/views/gcp/gcp.scala.html

```diff
@@ -68,14 +68,22 @@ <h3>GCP Security Findings</h3>
                     <div class="collapsible-header" tabindex="22">
                     <i class="material-icons">keyboard_arrow_down</i>
                     <span class="iam-header__name">@projectName</span>
-                        <span class="icon-count">@findings.count(_.severity == "High")</span>
-                        <i class="material-icons red-text">error</i>
-                        <span class="icon-count">@findings.count(_.severity == "Medium")</span>
-                        <i class="material-icons orange-text">warning</i>
-                        <span class="icon-count">@findings.count(_.severity == "Low")</span>
-                        <i class="material-icons yellow-text">priority_high</i>
-                        <span class="icon-count">@findings.count(_.severity == "Unknown")</span>
-                        <i class="material-icons grey-text">not_listed_location</i>
+                        @if(findings.exists(_.severity == "High")) {
```

Comment:
if these strings (low/high) are used in more than one place we should probably turn them into case objects like this  https://github.com/guardian/frontend/blob/968f803d15847ec4ec60a0818506a9c7fcc075f5/common/app/model/Section.scala#L72

Reply:
Great idea, thank you @philmcmahon! 

They're not used more than once right now, but if this changes I'll be sure to add this pattern. Thank you!👍🏽